src/concrete/ml/sklearn/base.py

```diff
@@ -31,6 +30,9 @@
 from sklearn.linear_model import LinearRegression, LogisticRegression
 from sklearn.utils.validation import check_is_fitted
 
+# pylint: disable-next=ungrouped-imports
+from concrete import fhe as cp
```

Comment:
why not just use fhe.tag?

Reply:
because of some pylint magic 🙂 

that's obviously what I've done at the beginning, which requires us to call `from concrete import fhe`. But guess what, since we have this `fhe` parameter in our `predict` methods, pylint was throwing a `W0621: Redefining name 'fhe' from outer scope (line 33) (redefined-outer-name)` 🙂 (https://github.com/zama-ai/concrete-ml/actions/runs/7463742016/job/20309199710)

and I didn't want to have `from concrete.fhe import tag, zeros, array` because calling `zeros(labels.shape)` or `array(arr)` in or code feels a bit ambiguous !

Reply:
Makes sense thanks a lot!